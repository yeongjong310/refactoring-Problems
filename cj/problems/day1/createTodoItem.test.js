/**
 * @jest-environment jsdom
 */

import { createTodoItem } from "./createTodoItem.js";

test("createTodoItem('dom test 해보기')", () => {
  expect(createTodoItem("dom test 해보기")).toMatchSnapshot();
});
