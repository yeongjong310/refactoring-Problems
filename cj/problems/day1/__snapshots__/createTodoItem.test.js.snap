// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createTodoItem('dom test 해보기') 1`] = `
<li
  class="todo__item"
>
  <div
    class="todo__item--wrapper"
  >
    <button
      class="todo__item--delete-btn"
    >
      <i
        class="fas fa-trash-alt"
      />
    </button>
    <span
      class="todo__item--title"
    >
      dom test 해보기
    </span>
  </div>
  <div
    class="todo__item--divider"
  />
</li>
`;
